2021-04-15 05:18:53,100 INFO    MainThread:19563 [wandb_setup.py:_flush():69] setting env: {}
2021-04-15 05:18:53,101 INFO    MainThread:19563 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-15 05:18:53,101 INFO    MainThread:19563 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210415_051853-2s4jakyy/logs/debug.log
2021-04-15 05:18:53,102 INFO    MainThread:19563 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210415_051853-2s4jakyy/logs/debug-internal.log
2021-04-15 05:18:53,102 INFO    MainThread:19563 [wandb_init.py:init():369] calling init triggers
2021-04-15 05:18:53,102 INFO    MainThread:19563 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-15 05:18:53,102 INFO    MainThread:19563 [wandb_init.py:init():418] starting backend
2021-04-15 05:18:53,102 INFO    MainThread:19563 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-15 05:18:53,150 INFO    MainThread:19563 [backend.py:ensure_launched():123] starting backend process...
2021-04-15 05:18:53,184 INFO    MainThread:19563 [backend.py:ensure_launched():127] started backend process with pid: 19636
2021-04-15 05:18:53,186 INFO    MainThread:19563 [wandb_init.py:init():423] backend started and connected
2021-04-15 05:18:53,189 INFO    MainThread:19563 [wandb_init.py:init():465] updated telemetry
2021-04-15 05:18:53,190 INFO    MainThread:19563 [wandb_init.py:init():484] communicating current version
2021-04-15 05:18:54,920 INFO    MainThread:19563 [wandb_init.py:init():489] got version response 
2021-04-15 05:18:54,921 INFO    MainThread:19563 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-15 05:18:55,074 INFO    MainThread:19563 [wandb_init.py:init():522] starting run threads in backend
2021-04-15 05:18:58,438 INFO    MainThread:19563 [wandb_run.py:_console_start():1524] atexit reg
2021-04-15 05:18:58,443 INFO    MainThread:19563 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-15 05:18:58,444 INFO    MainThread:19563 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-15 05:18:58,447 INFO    MainThread:19563 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-15 05:18:58,447 INFO    MainThread:19563 [wandb_init.py:init():546] run started, returning control to user process
2021-04-15 05:21:11,708 INFO    MainThread:19563 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-15 05:21:11,714 INFO    MainThread:19563 [wandb_run.py:_restore():1466] restore
2021-04-15 05:21:14,572 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 555
}

2021-04-15 05:21:15,153 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 962
}

2021-04-15 05:21:15,472 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:15,575 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:15,678 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:15,782 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:15,884 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:15,989 INFO    MainThread:19563 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5897
  total_bytes: 5897
}

2021-04-15 05:21:17,419 INFO    MainThread:19563 [wandb_run.py:_show_files():1838] logging synced files
